docs/faq.rst

```diff
@@ -230,7 +265,9 @@ Note:
 
 2. APIC minimum version supported for 4.1.x: 4.1(1k)
 
-Note: For L4-L7 App Services tab to get enabled, minimum AS3 plugin version required is 3.16
+3. APIC minimum version supported for 5.0.x: 5.0(1k)
+
+Note: For L4-L7 App Services tab to get enabled, minimum AS3 plugin version required is 3.19.1
```

Comment:
To enable the L4-L7 App services tab, you must be using the AS3 plugin version 3.19.1 or later.

(and probably should have mentioned this earlier, but AS3 isn't really a 'plugin' it's an extension, but probably ok to leave I suppose)

Reply:
Got it - will avoid referring to AS3 as AS3 plugin.